### DISCUSSION TITLE
스프린트3 기능

### DISCUSSION THREAD

**Discussion by Byeongju-Kong** [woowacourse-teams/2022-sokdak#157] [Ideas]
스프린트 3에서는 어떤 기능을 가져가면 좋을까요? 
우선 순위 순서로 적어주세요~!

**Comment by Byeongju-Kong** [comment]
1. 게시판 분리 (포수타, 감동크루, 자유게시판)
2. 로그인 되지 않으면 좋아요, 댓글 막기
3. 댓글 수정, 삭제
4. 좋아요 순서대로 보기
5. 키워드 검색
6. 해시태그 검색

**Comment by injoon2019** [comment]
1. 댓글 삭제
2. 게시판 분리
3. 검색 기능
4. 어드민 계정 (모든 글 삭제 가능)
5. 신고 기능
6. 내가 쓴 글/댓글 모아보기

**Comment by BETTERFUTURE4** [comment]
1. 자동 로그인 기능
2. 댓글 수정/삭제 기능
3. 게시판 분리
4. 상위 태그 노출/필터링 기능
![image](https://user-images.githubusercontent.com/50815519/180682126-39182d15-8759-40bf-ae1c-8d372a3656ce.png)
5. 베스트 글 선정 기능
6. 내 글 조회하기 기능
7. 글 검색 기능

---

8. 게시물 신고 기능
9. 댓글 추천 기능
10. 사진 업로드 기능
11. 베스트 댓글 선정 기능

**Comment by hyunrrr** [comment]
댓글 삭제
댓글 좋아요
개시판 분리
대댓글 cr

**Comment by byhhh2** [comment]
게시판 분리
해시태그 분류 (클릭시 해시태그 관련 글 목록으로 이동)
대댓글, 댓글 좋아요
베스트 댓글, 베스트 게시물

**Comment by EastHShin** [comment]
1. refresh token 구현을 통한 로그아웃/자동로그인 기능
2. 프론트 CI/CD와 dev서버와 운영서버 분리
3. 게시판 분리
4. 베스트 게시물
5. 해시태그를 통한 게시글 모아보기

**Comment by JUDONGHYEOK** [comment]
- 게시판 분리
- 프로필 페이지
- 검색기능
